build/storage/tests/it/ptf_tests/tests/test_hot_plug.py

```diff
@@ -63,6 +63,23 @@ def runTest(self):
             self.host_target_platform.get_number_of_virtio_blk_devices(), 0
         )
 
+        second_delete_responses = self.ipu_storage_platform.delete_virtio_blk_devices(
+            devices_handles
+        )
+        for response in second_delete_responses:
+            self.assertEqual(response, "False")
+
+        fake_devices_handles = self.ipu_storage_platform.create_fake_virtio_blk_devices(
+            self.host_target_platform.get_service_address(),
+            remote_nvme_storages,
+            range(len(remote_nvme_storages)),
+        )
+        fake_delete_responses = self.ipu_storage_platform.delete_virtio_blk_devices(
+            fake_devices_handles
+        )
+        for response in fake_delete_responses:
+            self.assertEqual(response, "False")
+
```

Comment:
Commit header is too long. Would it make sense to separate it to 2?

Reply:
when I will correct all issues I split this pr

Reply:
done